packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs

```diff
@@ -134,7 +136,9 @@
                   SourceFile="Redist\VCRedist_2015\vc_redist.x64.exe"
                   DownloadUrl="https://download.microsoft.com/download/6/3/a/63abaf83-2ca6-4460-90d7-12de8e815b1a/vc_redist.x64.exe"
                   >
-        <ExitCode Behavior="scheduleReboot"/>
+        <ExitCode Value="1641" Behavior="scheduleReboot"/>
+        <ExitCode Value="3010" Behavior="scheduleReboot"/>
+        <ExitCode Value="0" Behavior="success" />
```

Comment:
Should we add a comment or something to remind future us to remember to keep these two lists in sync?